lib/manager/docker-compose/extract.ts

```diff
@@ -79,9 +90,6 @@ export function extractPackageFile(
       .filter(Boolean);
 
     logger.trace({ deps }, 'Docker Compose image');
-    if (!deps.length) {
```

Comment:
The getPkgReleases() logic in datasource/index will actually convert an empty file to null anyway. We generally prefer to ignore files with no dependencies.